apps/slack/slack_listeners.py

```diff
@@ -61,10 +61,11 @@ def respond_to_message(event, context: BoltContext, session=None):
 
     slack_user = event.get("user")
 
+    experiment_version = experiment_channel.experiment.default_version
     if not session:
         external_id = make_session_external_id(channel_id, thread_ts)
         session = SlackChannel.start_new_session(
-            experiment_channel.experiment, experiment_channel, slack_user, session_external_id=external_id
+            experiment_version, experiment_channel, slack_user, session_external_id=external_id
```

Comment:
I think this should be the working version

Reply:
See https://github.com/dimagi/open-chat-studio/pull/671#discussion_r1754850320

Reply:
Same as reply to other comment

Reply:
https://github.com/dimagi/open-chat-studio/pull/671/commits/40f779b2f3819d6fa42a7426ab0177d2530e9611